### ISSUE TITLE
[BUG] cannot go get -u github.com/motemen/ghq 

### ISSUE THREAD

**Issue by kazu** [motemen/ghq#105]
```
% go get -u  github.com/motemen/ghq                     
# github.com/motemen/ghq/utils
.gvm/pkgsets/go1.10/global/src/github.com/motemen/ghq/utils/log.go:10:19: too few values in struct initializer
```

**Comment by kawakami-o3** [comment]
Use new constructor for logger #104

**Comment by motemen** [comment]
Thanks, shoud be fixed by merging #104.
